prebid-mobile/code-integration-android.md

```diff
@@ -164,10 +219,10 @@ if(conditionToStopRefresh) {
 }
  ```
 
-## If proguard is ON
+### ProGuard Support
 
-### Primary Ad Server is DFP
-To avoid dfp class being obfuscated and prebid not working, add the following lines to your proguard file:
+#### Primary Ad Server is DFP
+If ProGuard is enabled, the following lines must be added to your ProGuard file:
```

Comment:
@anwzhang Just a note that this is going to conflict with https://github.com/prebid/prebid.github.io/pull/891/files